src/test/software/amazon/event/ruler/CIDRTest.java

```diff
@@ -61,39 +61,40 @@ public void testToStringFailures() {
     @Test
     public void TestDigitSequence() {
 
-        List<Byte> l = Range.digitSequence((byte) '4', (byte) 'C', false, false);
-        byte[] wanted = { '5', '6', '7', '8', '9', 'A', 'B' };
+        byte[] l = Range.digitSequence((byte) '4', (byte) 'C', false, false);
+        byte[] wanted = {'5', '6', '7', '8', '9', 'A', 'B'};
         for (int i = 0; i < wanted.length; i++) {
-            assertEquals(wanted[i], (byte) l.get(i));
```

Comment:
Since I work mostly in Go these days, I really like absolutely unified formatting.  What tool are you using, and maybe we should put in a GitHub action or something to iron out the formatting.  In the IntelliJ Go IDE it now runs gofmt on every save :)

Reply:
intellij idea with default format, but yep I have notice very small differences vs the format, some unintentional reformats still slipping tho jaja, agree that would be nice to have a format enforced by the project so we dont have to care about them

Reply:
Added an issue for this https://github.com/aws/event-ruler/issues/43